contrib/pylightning/setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name='pylightning',
-      version='0.0.1',
+      version='0.0.2',
```

Comment:
This change seems unrelated?

Reply:
It is yes, I'll split it out and batch it with some other pylightning changes.